android/src/main/kotlin/com/onegini/mobile/sdk/flutter/useCases/CancelCustomRegistrationActionUseCase.kt

```diff
@@ -0,0 +1,24 @@
+package com.onegini.mobile.sdk.flutter.useCases
+
+import com.onegini.mobile.sdk.flutter.OneWelcomeWrapperErrors.IDENTITY_PROVIDER_NOT_FOUND
+import com.onegini.mobile.sdk.flutter.handlers.CustomRegistrationHandler
+import com.onegini.mobile.sdk.flutter.helpers.SdkError
+import com.onegini.mobile.sdk.flutter.providers.CustomRegistrationAction
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+
+class CancelCustomRegistrationActionUseCase(private val customRegistrationActions: ArrayList<CustomRegistrationAction>) {
+  operator fun invoke(result: MethodChannel.Result, call: MethodCall) {
+    val idProvider: String? = call.argument("identityProviderId")
+
+    when (val action = CustomRegistrationHandler().getCustomRegistrationAction(idProvider, customRegistrationActions)) {
+      null -> SdkError(IDENTITY_PROVIDER_NOT_FOUND).flutterError(result)
```

Comment:
This error is not actually always an IDENTITY_PROVIDER_NOT_FOUND. It could be a REGISTRATION_NOT_IN_PROGRESS.